### DISCUSSION TITLE
在群晖使用rclone挂载alist的webdav只能读不能写入

### DISCUSSION THREAD

**Discussion by dengboyun** [AlistGo/alist#6665] [General]
在群晖使用rclone挂载alist的webdav只能读不能写入有解决方案吗
<img width="1341" alt="image" src="https://github.com/alist-org/alist/assets/10330781/a8e92984-c1d0-42a9-9345-c930ac45cc3a">
![1719569884299](https://github.com/alist-org/alist/assets/10330781/49747d7b-eddc-4c2f-ac14-422794fd55ea)

**Comment by pongfcnkl** [comment]
挂载不支持file station
